module/PowerShellEditorServices/Commands/Public/Clear-Host.ps1

```diff
@@ -0,0 +1,14 @@
+#
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+Microsoft.PowerShell.Management\Get-Item function:Clear-Host | Microsoft.PowerShell.Management\Set-Item function:__clearhost
+
+function Clear-Host {
+    [Alias('cls')]
```

Comment:
Also `clear`?

Reply:
`clear` is not an alias it's an application on non-Windows. Is it an alias on Windows?